pricing/v1/pricing.proto

```diff
@@ -28,10 +28,10 @@ service Pricing {
     };
   }
 
-  // Get pricing info
-  rpc GetPricingInfo(GetPricingInfoRequest) returns (GetPricingInfoResponse) {
+  // Get pricing
```

Comment:
Add WORK-IN-PROGRESS as a prefix to the description. Like so,
`WORK-IN-PROGRESS: Get cloud pricing information`